Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -428,6 +428,16 @@ std::optional<Protocol::CSS::PseudoId> InspectorCSSAgent::protocolValueForPseudo
         return Protocol::CSS::PseudoId::ScrollbarCorner;
     case PseudoId::Resizer:
         return Protocol::CSS::PseudoId::Resizer;
+    case PseudoId::ViewTransition:
```

Comment:
in order for these to appear in Web Inspector you'll need to add these to `WI.CSSManager.displayNameForPseudoId` as well

Reply:
https://github.com/WebKit/WebKit/pull/20892